.travis.yml

```diff
@@ -1,89 +1,63 @@
-sudo: required
+# sudo: required
 branches:
     only:
         - master
-        - potato-1490727
-cache:
-    pip: True
-    directories:
-      - node_modules
-      - downloads
 language: python
+python:
+    - "2.7"
 services:
     - docker
-    - mysql
-    - redis
 env:
     global:
-        - DATABASE_URL=mysql://root:@127.0.0.1:3306/kuma
-        - DJANGO_SETTINGS_MODULE=kuma.settings.travis
+        - UID=0
         - DOCKER_COMPOSE_VERSION=1.23.2
-        - ES_VERSION=6.7.1
-        - ES_DOWNLOAD_URL=https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.7.1.tar.gz
-        - PIPELINE_CLEANCSS_BINARY=$TRAVIS_BUILD_DIR/node_modules/.bin/cleancss
-        - PIPELINE_CLEANCSS_ARGUMENTS="-O1 --skip-rebase"
-        - PIPELINE_CSS_COMPRESSOR=kuma.core.pipeline.cleancss.CleanCSSCompressor
-        - PIPELINE_JS_COMPRESSOR=pipeline.compressors.uglifyjs.UglifyJSCompressor
-        - PIPELINE_SASS_BINARY=$TRAVIS_BUILD_DIR/node_modules/.bin/node-sass
-        - PIPELINE_UGLIFYJS_BINARY=$TRAVIS_BUILD_DIR/node_modules/.bin/uglifyjs
+
+cache:
+  directories:
+    - $HOME/.cache/pip
+    - node_modules
 
 matrix:
     include:
-        - python: "2.7"
-          env:
-            TOXENV=pytest
-            CREATE_DB=kuma
-            INSTALL_ELASTICSEARCH=1
-            INSTALL_NPM_TOOLS=1
-        - python: "2.7"
-          env:
-            TOXENV=flake8
-        - python: "2.7"
-          env:
-            TOXENV=docs
-        - python: "2.7"
-          env:
-            TOXENV=locales
-            CREATE_DB=kuma
-            INSTALL_NPM_TOOLS=1
-        - python: "2.7"
-          env:
-            TOXENV=docker
-            INSTALL_DOCKER_COMPOSE=1
-            UID=0
-        - python: "2.7"
-          env:
-            TOXENV=assetlint
-            INSTALL_NPM_TOOLS=1
-        - python: "3.6"
-          env:
-            TOXENV=pytest3
-            CREATE_DB=kuma
-            INSTALL_ELASTICSEARCH=1
-            INSTALL_NPM_TOOLS=1
-    allow_failures:
-        - python: "3.6"
-          env:
-            TOXENV=pytest3
-            CREATE_DB=kuma
-            INSTALL_ELASTICSEARCH=1
-            INSTALL_NPM_TOOLS=1
-install:
-    - nvm install 10
-    - nvm use 10
-    - scripts/travis-install
-    - pip install -U pip
-    - pip install -r requirements/travis.txt
-    # Wait for ElasticSearch to be ready
-    - if [[ "$INSTALL_ELASTICSEARCH" == "1" ]]; then wget -q --waitretry=1 --retry-connrefused -T 10 -O - http://127.0.0.1:9200; fi;
-script:
-    - tox -v
-after_failure:
-    - dmesg | tail
-notifications:
-    irc:
-        channels:
-            - "irc.mozilla.org#mdndev"
-        on_success: always
-        on_failure: always
-        use_notice: true
+        - language: python
+          python: 3.6
+          # env:
+          #   BUILD_DOCS=1
+          #   # BUILD_DOCKER=0
+          #   # JSLINT=0
+          script:
```

Comment:
This was something that yesterday I was wondering if we could do (i.e., could each section of the build matrix have its own script). Nice!

Now that I know this, I would suggest we dump the bash scripts and just put the commands in their proper `script:` section in the build matrix, so that it's all in one place. I think we can do that since you can embed the `set -e` commands within the `script:` section as desired.

Reply:
I disagree. I much prefer doing the work inside a dedicated bash script. The yaml indentation is just a pain and editing a bash script gives my syntax highlighting for bash and not for yaml. 

I acknowledge that the file is named `travis-python` (for example) but as it stands, its *content* is theoretically agnostic of TravisCI. I.e. we *could* switch to CircleCI and keep this file as is.

Reply:
Actually, perhaps the name of the bash script(s) should be something like `ci-docs` instead of `travis-docs`.

Reply:
👍 Good points, let's stick with the bash scripts, and I'd vote for `ci-xxx` over `travis-xxx`, but whatever you prefer. It's fine the way it is as well, and we can change it if we ever change our CI provider.

Reply:
Want me to change that now?

Reply:
It would be nice, but honestly whatever you prefer.